docs/navigate/devops-testing/toc.yml

```
@@ -559,16 +559,22 @@ items:
         href: ../../core/rid-catalog.md
       - name: Resource manifest names
         href: ../../core/resources/manifest-file-names.md
-      - name: Docker
+      - name: Containers
         items:
+        - name: Overview
+          href: ../../core/containers/overview.md
+        - name: .NET SDK publish container
+          href: ../../core/containers/sdk-publish.md
+        - name: .NET SDK publish configuration
+          href: ../../core/containers/publish-configuration.md
```

```suggestion
        - name: .NET SDK publishing tutorial
          href: ../../core/containers/sdk-publish.md
        - name: .NET SDK publishing reference
          href: ../../core/containers/publish-configuration.md
```

maybe making it slightly more clear the split between tutorial and reference docs?